run.py

```diff
@@ -100,8 +101,10 @@
 
     if args.train:
         db.download()
-
-        model = model_class(args.lemmatization)
+        if args.historical and model_class == BugModel:
```

Comment:
Remove the second check here, if somebody passes `--historical` for an unsupported model, the script will just fail.